content/cpp/concepts/math-functions/terms/trunc/trunc.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'trunc()' 
+Description: 'Returns the integral part of the argument.' 
+Subjects:
+  - 'Computer Science'
+Tags: 
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`trunc()`** function returns the integral part of the argument. It rounds the argument towards zero and returns the nearest integer that is not greater in magnitude than the argument.
+
+## Syntax
+
+```cpp
+std::trunc(x);
+```
+
+## Example
+
+```cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+    double x = 3.14159;
+    double result;
+
+    result = std::trunc(x);
+
+    std::cout << "trunc(" << x << ") = " << result << "\n";
+    // Output: trunc(3.14159) = 3
+}
+```
+
+## Codebyte Example
+
+Use `trunc()` to round the double `12.3456` down to the nearest integer by removing the decimal part:
```

Comment:
The phrasing here is misleading since the `trunc()` function doesn't round to the nearest integer (though it does in this case) but strips off the bit after the decimal. (i.e. if the argument was 12.9, the result would still be 12.)